AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -143,20 +143,30 @@ internal class CardViewController: FormViewController {
             items.numberItem.showLogos(for: brands.map(\.type))
         }
 
-        let isHidden: Bool
+        let kcpItemsHidden: Bool
         switch configuration.koreanAuthenticationMode {
         case .show:
-            isHidden = false
+            kcpItemsHidden = false
         case .hide:
-            isHidden = true
+            kcpItemsHidden = true
         case .auto:
-            isHidden = !configuration.showAdditionalAuthenticationFields(for: binInfo.issuingCountryCode)
+            kcpItemsHidden = !configuration.showAdditionalAuthenticationFields(for: binInfo.issuingCountryCode)
+        }
+        
+        var socialSecurityItemHidden: Bool
```

Comment:
can we move this to a separate `var`? like this:

```Swift
var socialSecurityItemHidden: Bool {
switch configuration.socialSecurityNumberMode {
         case .show:
             socialSecurityItemHidden = false
         case .hide:
             socialSecurityItemHidden = true
         case .auto:
             socialSecurityItemHidden = !brands.socialSecurityNumberRequired
         }
}
```

and also `kcpItemIsHidden`?

Reply:
Hmm, okay then I will move them and rename both to 'shouldHide....` syntax. What do you think?

Reply:
yeah, `shouldHide...` sounds good!

Reply:
Can't make them vars as they require bin response. Making them functions just for this seems redundant. Shall we just keep it like this then?